### PR TITLE
storage: collect flow statistics in batch_get_commands

### PR DESCRIPTION
**Pull request by sticnarf**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/pd/issues/2499

Problem Summary:

Sometimes PD receives zero read bytes when running `get` or `batch_get`.

When `enable_batch_commands` is `true`, TiKV packs several get requests into a `batch_get_commands` if load is high. However, there was a mistake that the flow statistics of `batch_get_commands` are not collected. 

### What is changed and how it works?

What's Changed:

Collect statistics in `batch_get_commands`.

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Previously, although there was lots of point selects, read bytes keep 0.
image
image

This PR:
image

image

### Release note 

Fix the bug that read statistics of point selects are inaccurate.

### CODE REVIEW

**Comment by youjiali1995** [comment]
...

**Comment by sticnarf** [comment]
Wait a minute, let me also add the collection for raw_batch_commands

File: src/storage/mod.rs [hunk 1]
```diff
@@ -331,6 +331,8 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
                                 .map_err(Error::from),
                         );
                     }
+                    metrics::tls_collect_scan_details(CMD, &statistics);
+                    metrics::tls_collect_read_flow(ctx.get_region_id(), &statistics);
```
Comment (lhy1024) [inline src/storage/mod.rs:335]: we also need to add collection for `raw_batch_get_command`

**Comment by sticnarf** [comment]
(@AndreMouche wants to debug the sre-bot)

**Pull request merged**